hack/build/cleanup_docker.sh

```diff
@@ -1,6 +1,10 @@
-#!/bin/bash
-set -e
+#!/usr/bin/env bash
```

Comment:
why the first line is changed?

Reply:
I've left it alone, but /usr/bin/env bash doesn't assume the location of the bash installation,
s/set -e/set -ex/ is a debug leftover - removed